inheritance/src/main/java/inheritance/Restaurant.java

```diff
@@ -0,0 +1,41 @@
+package inheritance;
+
+import java.util.LinkedList;
+
+public class Restaurant implements Reviewable{
+
+    String restaurantName;
+    int stars;
+    String price;
+    LinkedList<Review> restaurantReviews;
+
+//    restaurant constructor
+    public Restaurant (String restaurantName, int stars, String price){
+
+        this.restaurantName = restaurantName;
+        this.stars = stars;
+        this.price = price;
+        this.restaurantReviews = new LinkedList<>();
+    }
+
+    @Override
+    public void addReview(Review review) {
+//        addReview adds a review to the LL using .add builtin method
+//        addReview is going to add pointer (which helps the communication between interface)
+
+        restaurantReviews.add(review);
+//        ?????
```

Comment:
`review.business = this`: Your class, Review, has an instance variable called business. Line 28 points business towards the Restaurant that the review is about.